src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,51 @@
+package duke.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Event class represents a task with a specific start and end time in the Duke chatbot application.
+ * It extends the Task class.
+ */
+public class Event extends Task {
+
+    private static final String EVENT_MESSAGE = "[E]%s (from: %s to: %s)";
+    private static final String EVENT_FILE_TEMPLATE = "E | %s | %s | %s";
```

Comment:
I liked how you stored the template as constants. Perhaps the icon `E` can be stored as another constant as it is duplicated?